raco/representation.py

```diff
@@ -0,0 +1,17 @@
+__author__ = 'brandon'
+
+
+class RepresentationProperties(object):
+    def __init__(self, hash_partitioned=set(), sorted=None, grouped=None):
+        """
+        @param hash_partitioned: None or list of AttributeRefs in hash key
```

Comment:
Unrelated: initializing `hash_partitioned` with a mutable default argument (evaluated at time of function definition) is probably not what you intended.